src/Core/src/Graphics/Ellipse.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Graphics;
+
+namespace Microsoft.Maui.Graphics
+{
+	public class Ellipse : Shape
```

Comment:
Let's consider adding some extension methods to the Path or PathBuilder in Maui Graphics and not requiring concrete types of shapes in Core.  We can always add Shape and Ellipse, etc to Controls if it is necessary and valuable there.